packages/yoroi-connector/src/inject.js

```diff
@@ -33,10 +33,10 @@ async function injectIntoPage(code) {
         container.removeChild(scriptTag);
         console.log(`[yoroi/${YOROI_TYPE}] dapp-connector is successfully injected into ${location.hostname}`);
         markInjectionInDocument(container);
-        return true;
+        resolve(true);
```

Comment:
This change is problematic. I forgot to remove the `return true` statement. But the promise should not be resolved here. See `resolveScriptedInject`.

Reply:
ahhh, understood, I will update 👌